### PR TITLE
Focal

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/gir-files/pull/52

cc @GuillaumeGomez , @sdroege

### CODE REVIEW

**Comment by EPashkin** [comment]
Updated to https://github.com/gtk-rs/gir-files/pull/55
Contains two commits, changes before focal from git update and focal changes

File: Gir.toml [hunk 1]
```diff
@@ -2064,6 +2060,9 @@ status = "generate"
     name = "get_model"
         [object.function.return]
         nullable = false
+    [[object.function]]
+    name = "new_with_model"
+    rename = "new"
```
Comment (sdroege) [inline Gir.toml:2065]: Why? There's also a normal `new()` that's useful by itself and works differently

Comment (sdroege) [reply Gir.toml:2065]: Oh there is no `new()` and because that's how we had it in manual code before. Why?

Comment (EPashkin) [reply Gir.toml:2065]: Before this version `gtk_tree_model_sort_new_with_model` was function in gtk_tree, so we added call for it in `TreeModelSort` for better usability, now it moved to `TreeModelSort` in upstream, so we just rename it in Rust part

Comment (sdroege) [reply Gir.toml:2065]: Ok :man_shrugging:

**Comment by EPashkin** [comment]
Updated

**Comment by sdroege** [comment]
Looks good to me, let's merge after sys is done :)

**Pull request merged**